packages/@dcl/inspector/src/components/EntityInspector/ActionInspector/ActionInspector.tsx

```diff
@@ -61,14 +57,13 @@ const ActionMapOption: Record<string, string> = {
 }
 
 export default withSdk<Props>(
-  withContextMenu<Props & WithSdkProps>(({ sdk, entity: entityId, contextMenuId }) => {
+  withContextMenu<Props & WithSdkProps>(({ sdk, entity: entityId }) => {
```

Comment:
do we need the `withContextMenu` higher order component if we are not using the `contextMenuId` prop?